### PR TITLE
fix Growth in Dynamic Linear Economies

### PR DESCRIPTION
**Pull request by fkazemian**

### CODE REVIEW

**Comment by jstac** [comment]
Thanks @fkazemian 

@natashawatkins , did you have time to look this over?

File: source/rst/growth_in_dles.rst [hunk 1]
```diff
@@ -451,137 +447,108 @@ These preferences reflect habit persistence
...
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+========================================
 
-.. code:: ipython3
+.. code-block:: python3
 
-    beta2 = np.array([[0.94]])
-    Pref4 = (beta2,llambda,pih,deltah,thetah)
+    β2 = np.array([[0.94]])
+    Pref4 = (β2,lλ,πh,δh,θh)
```
Comment (natashawatkins) [inline source/rst/growth_in_dles.rst:530]: @fkazemian could you please add spaces after commas here

File: source/rst/growth_in_dles.rst [hunk 2]
```diff
@@ -1,84 +1,106 @@
...
 Three examples of such economies show how the DLE class can be used to
 compute equilibria of such economies in Python and to illustrate how
 different versions of these economies can or cannot generate sustained
-growth.
+growth
+
+
+We will need the following imports
+
+.. code-block:: python3
```
Comment (natashawatkins) [inline source/rst/growth_in_dles.rst:31]: this one should be `ipython` because it has a magic

**Comment by thomassargent30** [comment]
Hi. Natasha. It seems like people have been editing this weekend, but I see
nothing new when I pull.  Any suggestions?

thanks, tom

**Comment by natashawatkins** [comment]
Are you on the master branch? There should be new edits from 2 days ago. But maybe you already pulled then.

**Comment by thomassargent30** [comment]
I have the ones from Friday.  John says they are the latest. I am on master
branch.

**Comment by natashawatkins** [comment]
@thomassargent30 I just merged this one in so please feel free to pull and work on the growth lecture now

**Pull request merged**